### ISSUE TITLE
Crash when revealing points

### ISSUE THREAD

**Issue by gcrickman** [ParabolInc/parabol#4807]
## Issue - Bug
A [user](https://app.hubspot.com/contacts/3888472/ticket/322230317/) 🔒  writes:

> The biggest issue was every time the points should be revealed, every team member got a crash message:

![image](https://user-images.githubusercontent.com/60167404/112214243-67a73280-8c38-11eb-942a-ff06714e65a1.png)

> I've attached the console log as well from this crash. 

```
Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at https://cdn.segment.com/analytics.js/v1/sZWp1xJSUiw3T5LSqGDGmbzGKBlOx68y/analytics.min.js. (Reason: CORS request did not succeed).
<script> source URI is not allowed in this document: â€œhttps://cdn.segment.com/analytics.js/v1/sZWp1xJSUiw3T5LSqGDGmbzGKBlOx68y/analytics.min.jsâ€. 4:1:1
TypeError: e is undefined
    bn PokerDiscussVoting.tsx:76
    bn PokerDiscussVoting.tsx:75
    React 6
    unstable_runWithPriority scheduler.production.min.js:19
    React 6
    r ReactRelayFragmentContainer.js:65
    e RelayModernFragmentSpecResolver.js:69
    e RelayModernFragmentSpecResolver.js:205
    _updateSubscription RelayModernStore.js:394
    notify RelayModernStore.js:241
    notify RelayModernStore.js:240
    p RelayProfiler.js:142
    run RelayPublishQueue.js:197
    _processResponse RelayModernQueryExecutor.js:452
    _handleNext RelayModernQueryExecutor.js:280
    _next RelayModernQueryExecutor.js:220
    _schedule RelayModernQueryExecutor.js:191
    _next RelayModernQueryExecutor.js:219
    next RelayModernQueryExecutor.js:106
    next RelayObservable.js:535
    dispatch index.js:173
    <anonymous> index.js:140
    emit index.js:202
    onmessage index.js:407
instrument.js:110:45
TypeError: e is undefined
    bn PokerDiscussVoting.tsx:76
    bn PokerDiscussVoting.tsx:75
    React 6
    unstable_runWithPriority scheduler.production.min.js:19
    React 6
    r ReactRelayFragmentContainer.js:65
    e RelayModernFragmentSpecResolver.js:69
    e RelayModernFragmentSpecResolver.js:205
    _updateSubscription RelayModernStore.js:394
    notify RelayModernStore.js:241
    notify RelayModernStore.js:240
    p RelayProfiler.js:142
    run RelayPublishQueue.js:197
    _processResponse RelayModernQueryExecutor.js:452
    _handleNext RelayModernQueryExecutor.js:280
    _next RelayModernQueryExecutor.js:220
    _schedule RelayModernQueryExecutor.js:191
    _next RelayModernQueryExecutor.js:219
    next RelayModernQueryExecutor.js:106
    next RelayObservable.js:535
    dispatch index.js:173
    <anonymous> index.js:140
    emit index.js:202
    onmessage index.js:407
instrument.js:110:45
Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at https://sentry.io/api/107196/store/?sentry_key=11a92d26438147a2a00805494c070f3b&sentry_version=7. (Reason: CORS request did not succeed).
TypeError: e is undefined
    bn PokerDiscussVoting.tsx:76
    bn PokerDiscussVoting.tsx:75
    React 6
    unstable_runWithPriority scheduler.production.min.js:19
    React 6
    r ReactRelayFragmentContainer.js:65
    e RelayModernFragmentSpecResolver.js:69
    e RelayModernFragmentSpecResolver.js:205
    _updateSubscription RelayModernStore.js:394
    notify RelayModernStore.js:241
    notify RelayModernStore.js:240
    p RelayProfiler.js:142
    run RelayPublishQueue.js:197
    _processResponse RelayModernQueryExecutor.js:452
    _handleNext RelayModernQueryExecutor.js:280
    _next RelayModernQueryExecutor.js:220
    _schedule RelayModernQueryExecutor.js:191
    _next RelayModernQueryExecutor.js:219
    next RelayModernQueryExecutor.js:106
    next RelayObservable.js:535
    dispatch index.js:173
    <anonymous> index.js:140
    emit index.js:202
    onmessage index.js:407
instrument.js:110:45
    be instrument.js:110
    f ErrorComponent.tsx:28
    React 7
    unstable_runWithPriority scheduler.production.min.js:19
    React 6
    r ReactRelayFragmentContainer.js:65
    e RelayModernFragmentSpecResolver.js:69
    e RelayModernFragmentSpecResolver.js:205
    _updateSubscription RelayModernStore.js:394
    notify RelayModernStore.js:241
    notify RelayModernStore.js:240
    p RelayProfiler.js:142
    run RelayPublishQueue.js:197
    _processResponse RelayModernQueryExecutor.js:452
    _handleNext RelayModernQueryExecutor.js:280
    _next RelayModernQueryExecutor.js:220
    _schedule RelayModernQueryExecutor.js:191
    _next RelayModernQueryExecutor.js:219
    next RelayModernQueryExecutor.js:106
    next RelayObservable.js:535
    dispatch index.js:173
    <anonymous> index.js:140
    emit index.js:202
    onmessage index.js:407
TypeError: e is undefined
    bn PokerDiscussVoting.tsx:76
    bn PokerDiscussVoting.tsx:75
    React 6
    unstable_runWithPriority scheduler.production.min.js:19
    React 6
    r ReactRelayFragmentContainer.js:65
    e RelayModernFragmentSpecResolver.js:69
    e RelayModernFragmentSpecResolver.js:205
    _updateSubscription RelayModernStore.js:394
    notify RelayModernStore.js:241
    notify RelayModernStore.js:240
    p RelayProfiler.js:142
    run RelayPublishQueue.js:197
    _processResponse RelayModernQueryExecutor.js:452
    _handleNext RelayModernQueryExecutor.js:280
    _next RelayModernQueryExecutor.js:220
    _schedule RelayModernQueryExecutor.js:191
    _next RelayModernQueryExecutor.js:219
    next RelayModernQueryExecutor.js:106
    next RelayObservable.js:535
    dispatch index.js:173
    <anonymous> index.js:140
    emit index.js:202
    onmessage index.js:407
instrument.js:110:45
TypeError: e is undefined
    bn PokerDiscussVoting.tsx:76
    bn PokerDiscussVoting.tsx:75
    React 6
    unstable_runWithPriority scheduler.production.min.js:19
    React 6
    r ReactRelayFragmentContainer.js:65
    e RelayModernFragmentSpecResolver.js:69
    e RelayModernFragmentSpecResolver.js:205
    _updateSubscription RelayModernStore.js:394
    notify RelayModernStore.js:241
    notify RelayModernStore.js:240
    p RelayProfiler.js:142
    run RelayPublishQueue.js:197
    _processResponse RelayModernQueryExecutor.js:452
    _handleNext RelayModernQueryExecutor.js:280
    _next RelayModernQueryExecutor.js:220
    _schedule RelayModernQueryExecutor.js:191
    _next RelayModernQueryExecutor.js:219
    next RelayModernQueryExecutor.js:106
    next RelayObservable.js:535
    dispatch index.js:173
    <anonymous> index.js:140
    emit index.js:202
    onmessage index.js:407
instrument.js:110:45
```

**Comment by tianrunhe** [comment]
I think this should already be fixed by [my PR](https://github.com/ParabolInc/parabol/pull/4747).

**Comment by jordanh** [comment]
Closing since #4747 was merged; should make it into the next release (v6.5.0)
